source/deployment/cluster.rst

```diff
@@ -1,116 +1,126 @@
+3. Modify the ``config.json`` files on both servers to add the ``ClusterSettings`` as described in the *High Availability (Beta)* section of :doc:`../administration/config-settings`.
 4. Verify the configuration files are identical on both servers then restart each machine in the cluster.
-5. `Modify your NGINX setup <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ so it proxies to both servers.
-6. Open the **System Console** > **Advanced** > **High Availability (Beta)** to verify all the machines in the cluster are communicating as expected with green status indicators. If not, investigate the log files for any extra information.
+5. Modify your NGINX setup  so that it proxies to both servers. For more information about this, see `Proxy Server Configuration`_
+6. Open **System Console > Advanced > High Availability (Beta)** to verify all the machines in the cluster are communicating as expected with green status indicators. If not, investigate the log files for any extra information.
 
 Adding a Server to the Cluster
-------------------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-1. Backup your `Mattermost database <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_ and `file storage location <https://docs.mattermost.com/deployment/cluster.html#file-storage-configuration>`_.
-2. Setup a new Mattermost server following one of our **Install Guides**. This server must use an identical copy of the configuration file, ``config.json``. Verify the server is functioning by hitting the private IP address.
-3. Modify the ``config.json`` files on all servers with the ``ClusterSettings`` as `described in this documentation <https://docs.mattermost.com/administration/config-settings.html#high-availability-beta>`_. Be sure to add the new server URL to ``InterNodeUrls``. 
-4. Verify the configuration files are identical on all servers, then restart each machine in the cluster in sequence with 10 or more seconds between each restart.
-5. `Modify your NGINX setup <https://docs.mattermost.com/deployment/cluster.html#proxy-server-configuration>`_ to add the new server.
-6. Open the **System Console** > **Advanced** > **High Availability (Beta)** to verify all the machines in the cluster are communicating as expected with green status indicators. If not, investigate the log files for any extra information.
+1. Backup your Mattermost database and the file storage location.
+2. Setup a new Mattermost server. This server must use an identical copy of the configuration file, ``config.json``. Verify the server is functioning by hitting the private IP address.
+3. Modify the ``config.json`` files on all servers with the ``ClusterSettings`` as described in the *High Availability (Beta)* section of :doc:`../administration/config-settings`. Be sure to add the new server URL to ``InterNodeUrls``. 
```

Comment:
I guess we're trying to avoid doing a direct link in case titles change and the links break? 

Eg https://docs.mattermost.com/administration/config-settings.html?highlight=high%20availability#high-availability-beta

At worst, if the titles change I think it just takes you to the top of the document, so it might be better than nothing